hyperbahn/clients/index.js

```diff
@@ -300,6 +307,18 @@ ApplicationClients.prototype.onRemoteConfigUpdate = function onRemoteConfigUpdat
     self.updateExemptServices();
     self.updateRpsLimitForServiceName();
     self.updateKValues();
+    self.updateReservoir();
+};
+
+ApplicationClients.prototype.updateReservoir = function updateReservoir() {
+    var self = this;
+    if (self.logReservoir) {
+        var size = self.remoteConfig.get('log.reservoir.size', 100);
```

Comment:
Wait we made a mistake here; we do not want the default to be `100`; let's bump the default to `1000`